### ISSUE TITLE
Dialog UX is cropped after zooming at 400%.

### ISSUE THREAD

**Issue by najeebrizvi** [microsoft/fluentui#20285]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

### Describe the issue:
Dialog UX is cropped after zooming at 400% zoom. The options in the dialog box becomes inaccessible in absence of horizontal scroll bars.
![image](https://user-images.githubusercontent.com/66671342/138282612-5c163884-4011-4ee5-b599-af53f58b2841.png)

### Please provide a reproduction of the issue in a codepen:
1. open the codpen https://codepen.io/najeebcodepen/pen/abymxXq
2. Launch the dialog
3. Zoon to the page to 400%, the dialog content is getting cropped.

#### Actual behavior:
1. The dialog content is not accessible.

#### Expected behavior:
1. Horizontal scrollbars should be there to make the dialog more accessible accessible.

**Comment by VenkateshChunchu** [comment]
We do facing the same issue. @gouttierre
